mlrun/runtimes/sparkjob/abstract.py

```diff
@@ -529,16 +549,36 @@ def _run(self, runobj: RunObject, execution: MLClientCtx):
                         gpu_quantity,
                         int,
                     )
+        driver_cpu_limit = None
         if "limits" in self.spec.driver_resources:
             if "cpu" in self.spec.driver_resources["limits"]:
+                driver_cpu_limit = self.spec.driver_resources["limits"]["cpu"]
                 verify_and_update_in(
                     job,
                     "spec.driver.coreLimit",
                     self.spec.driver_resources["limits"]["cpu"],
                     str,
                 )
         if "requests" in self.spec.driver_resources:
-            # CPU Requests for driver moved to child classes as spark3 supports string (i.e: "100m") and not only int
+            if "requests" in self.spec.driver_resources:
```

Comment:
seems like redundant "if" clause (same as above)

Reply:
Thanks, not sure how that happened.